Cargo.lock

```diff
@@ -10,7 +10,7 @@ dependencies = [
  "cocoa",
  "core-foundation 0.10.0",
  "objc",
- "thiserror 2.0.3",
```

Comment:
Revert changes to Cargo.lock, we don't want to update immediately before release.